assets/templates/partials/filter.tmpl

```diff
@@ -1,83 +1,94 @@
-                                           aria-haspopup="true"
-                                              {{ range $content := $category.ContentTypes}}
-                                                  {{if stringArrayContains $content.Group $filter }}
-                                                      checked
-                                                  {{end}}
-                                              {{end}}
-                                           >
-                                    <label class="ons-checkbox__label" for="group-{{ $index }}"
-                                           id="group-{{ $index }}-label">
-                                        {{ localise $category.LocaliseKeyName $lang 4 }} ({{ $category.Count }})
-                                    </label>
-                                    <span class="ons-checkbox__other {{ range $content := $category.ContentTypes }}{{ range $filter }}{{ if eq . $content.Group }}ons-checkbox__other--open{{end}}{{end}}{{end}}" id="group-{{ $index }}-other-wrap">
-                                        <fieldset class="ons-fieldset ons-js-other-fieldset">
-                                            {{ range $contentIndex, $content := $category.ContentTypes }}
-                                                <div class="ons-checkboxes__items">
-                                                <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
-                                                    <span class="ons-checkbox ons-checkbox--no-border child-filter">
-                                                        {{ $id := (print $content.Group) }}
-                                                        <input id="{{ $id }}"
-                                                               class="js-auto-submit__input ons-checkbox__input ons-js-checkbox"
-                                                               type="checkbox" name="filter"
-                                                               {{ range $filter }}{{ if eq . $content.Group }}checked{{end}}{{end}}
-                                                               value="{{ $id }}"
-                                                        />
-                                                        <label class="ons-checkbox__label" for="{{ $id }}"
-                                                               id="{{ $id }}-label">
-                                                            {{ localise $content.LocaliseKeyName $lang 4 }} ({{ $content.Count }})
-                                                        </label>
-                                                    </span>
+            </h2>
+            <span class="ons-collapsible__icon">
+                <svg class="ons-svg-icon " viewBox="0 0 8 13" xmlns="http://www.w3.org/2000/svg" focusable="false" fill="currentColor">
```

Comment:
We should probably add this icon as a template in the renderer to be consistent? 

See: https://github.com/ONSdigital/dp-renderer/tree/main/assets/templates/icons